docs/customosbuildinstructions.md

```diff
@@ -105,6 +105,12 @@ Here are the expected contents of each subdirectory /file
              /bin/iproute
              /bin/hostname
 
+    - Required binaires: utilities used by docker
+
+             /bin/ls
+             /bin/cat
+             /bin/test
```

Comment:
Oops, I was working a few other file and thinking it's a new binary that we generated. 
Please ignore my previous comments.